### PR TITLE
Home page copy updates

### PR DESCRIPTION
**Pull request by ryankeairns**
🚧 one more thing to review before this can be merged.

There are more changes in the pipeline, but our hope is that these copy/messaging changes will suffice for the time being (we've collected alot of consistent feedback that the current messaging is confusing). This new copy is based upon quite a bit of customer research/feedback... we feel confident that's hitting on the right concepts.

Regarding what's upcoming, we're working on a real-world metaphor that will likely have an even larger impact on the home page, but until then - these changes should help.

### CODE REVIEW

**Comment by thesentinels** [comment]
@ryankeairns, thanks for your PR! By analyzing the annotation information on this pull request, we identified @magwalk, @davidwrede, @TrevorBramble, @smith and @nathenharvey to be potential reviewers

File: www/source/index.html.slim [hunk 1]
```diff
@@ -8,17 +8,10 @@ section.hero
...
-            the application instead of the infrastructure it runs on.
-            With Habitat, the apps you build, deploy, and manage behave
-            consistently in any runtime&thinsp;&mdash;&thinsp;bare metal, VMs,
-            containers, and PaaS. You'll spend less time on the environment and
-            more time building features.
-        p.hero--text
-          | Habitat is an open source project, and we’d love for you to
-            get involved.
+          | Habitat enables you to ship your application, to any platform,
+            along with all the tools you'll need to manage it in production. And it's all open-source :)
```
Comment (TrevorBramble) [inline www/source/index.html.slim]: > And it's all open-source :)
Should probably include punctuation.

File: www/source/index.html.slim [hunk 2]
```diff
@@ -32,188 +25,195 @@ section.sub-hero
...
+          .sub-hero--blurb.columns.medium-3.medium-offset-1.end
+            .sub-hero--blurb-title
+              img.sub-hero--icon[src="/images/icons/subhero-config.svg"
+                onerror="this.src='/icons/subhero-config.png'" alt=""]
+              h4 Manage
             p
-              | Run fast, run smart with Habitat. Habitat apps are immutable
-                and atomically deployed, with self-organizing peer relationships.
+              | Once your app is running, use Habitat's built-in tooling to
+                manage it throughout its entire lifecycle thus minimizing
```
Comment (TrevorBramble) [inline www/source/index.html.slim]: Reads a little breathlessly without a comma between "lifecycle" and "thus".

File: www/source/index.html.slim [hunk 3]
```diff
@@ -32,188 +25,195 @@ section.sub-hero
...
+      h2.use-cases--title
+        | It's much more than a packaging system.
+      .use-cases
+        .row
+          .use-cases--wrap.columns.medium-5.medium-offset-1
+            .use-cases--box
+              h4 Instant portability
+              p
+                | Habitat artifacts are fully capable of running on their own or
+                  you can export them into a variety of images formats.
```
Comment (TrevorBramble) [inline www/source/index.html.slim]: > variety of image~~s~~ formats

File: www/source/index.html.slim [hunk 4]
```diff
@@ -32,188 +25,195 @@ section.sub-hero
...
-        p
-          | Habitat is unapologetically app-centric. It's designed with best
-            practices for the modern application in mind.
+      .production--blurb.columns.medium-3.medium-offset-1
+        h2 What's does Habitat provide?
         p
-          | Habitat gives you a packaging format and a runtime supervisor
-            with deployment coordination and service discovery built in.
+          | When you build a Habitat package you’re getting access to a host of
+            built-in features typically attained through a tangled mess of 3rd
```
Comment (TrevorBramble) [inline www/source/index.html.slim]: > ~~3rd party~~  third-party

imho =^)

File: www/source/index.html.slim [hunk 5]
```diff
@@ -32,188 +25,195 @@ section.sub-hero
...
+          p
+            | Each service starts with a supervisor process that takes commands
+              and communicates them amongst its peers.
+          a href="/docs/concepts-supervisor/" View docs
+        .production--features-item
+          img.production--icon[src="/images/icons/subhero-results.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Isolated Build Environment
+          p
+            | Define your entire app in a plan file and generate reproducble,
```
Comment (TrevorBramble) [inline www/source/index.html.slim]: > reproducble

File: www/source/index.html.slim [hunk 6]
```diff
@@ -32,188 +25,195 @@ section.sub-hero
...
+          a href="/docs/run-packages-monitoring/" View docs
 
+      .production--features.columns.medium-3.medium-offset-1
+        .production--features-item
+          img.production--icon[src="/images/icons/subhero-platform.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Runtime Support
+          p
+            | Configuration changes and files can be passed and circulated to
+              your app in a single command.
```
Comment (TrevorBramble) [inline www/source/index.html.slim]: > ~~in~~ with a single command

maybe?

File: www/source/index.html.slim [hunk 7]
```diff
@@ -32,188 +25,195 @@ section.sub-hero
...
+          a href="/docs/run-packages-overview/" View docs
+        .production--features-item
+          img.production--icon[src="/images/icons/subhero-manage.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Clustering Strategies
+          p
+            | Start up your services in a variety of built-in topologies such as
+              leader/follower, standalone and initializer.
```
Comment (TrevorBramble) [inline www/source/index.html.slim]: I'm not certain of what "initializer" means in this context, but if it's not something you pair with "standalone" there should be an Oxford comma here to separate them.

File: www/source/index.html.slim (hunk 1 above)
Comment (juliandunn) [inline www/source/index.html.slim]: I'm also not a huge fan of the `:)` as I think it's a little too cutesy.

**Comment by ryankeairns** [comment]
@TrevorBramble thanks for taking the time to read all the copy, very helpful edits!

File: www/source/index.html.slim (hunk 1 above)
Comment (ryankeairns) [inline www/source/index.html.slim]: :) has died in a 🔥 . Thank for the feedback!

**Comment by TrevorBramble** [comment]
@ryankeairns Don't know if you want to linger for more feedback so I'll hold the Sentinels. You've got my :+1: though!

**Comment by ryankeairns** [comment]
I'll hold this until Monday to see if any last minute feedback comes in via Slack, then we'll release the hounds 🐕

File: www/source/index.html.slim [hunk 8]
```diff
@@ -32,188 +26,195 @@ section.sub-hero
...
+          p
+            | All communications to and between your services are secured with
+              high-grade encryption.
+          a href="/docs/run-packages-security/" View docs
+        .production--features-item.last-item
+          img.production--icon[src="/images/icons/subhero-platform.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Service Discovery
+          p
+            | Connect running services into a ring, enabling them to
```
Comment (juliandunn) [inline www/source/index.html.slim]: how about just "Connect running services together"

File: www/source/index.html.slim [hunk 9]
```diff
@@ -32,188 +26,195 @@ section.sub-hero
...
+          .sub-hero--blurb.columns.medium-3.medium-offset-1
+            .sub-hero--blurb-title
+              img.sub-hero--icon[src="/images/icons/subhero-deploy.svg"
+                onerror="this.src='/icons/subhero-deploy.png'" alt=""]
+              h4 Deploy
             p
-              | Make your app production ready with monitoring, runtime
-                configuration, rolling deployment, and more.  Put an end to
-                "last-mile" configuration nightmares.
+              | Ship your Habitat build artifact as-is or export it into other
```
Comment (juliandunn) [inline www/source/index.html.slim]: I think if your build artifact is singular, the export format should be singular too, so it should be "export it into another format, like a container"

**Comment by ryankeairns** [comment]
All of the aforementioned edits have been accounted for up through this morning (Monday, Sep 19).

**Pull request merged**